ices/79018.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I don't think we should track this ICE here since it's _intended behavior_ and won't fix in the meanwhile (read the issue comments).

Reply:
I've been looking for an _intentional_ ICE ever since `break rust;` doesn't work anymore. This should do the trick nicely.